### PR TITLE
Add Cilium dev workflow to readme

### PR DESCRIPTION
**Pull request by joestringer**
This should probably be documented under Cilium documentation, but someone
asked about it so I figured I'd post it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +148,51 @@ go run ./cmd/lvh run --image _data/images/base.qcow2
...
+```bash
+./scripts/pull_image.sh quay.io/lvh-images/kind:bpf-next-main
+lvh run --image $(ls -Art images/*.qcow2 | tail -n 1) --host-mount /path/to/cilium
+```
+
+Then, log into the VM as root and setup some dependencies:
+
+```bash
+# DNS
+echo "nameserver 1.1.1.1" > /etc/resolv.conf
```
Comment (aditighag) [inline README.md:164]: These steps line up with the steps I had written down based on my local testing.

**Review by aditighag — COMMENTED** [review]
❤️ I haven't tried the cilium dev workflow yet, but it's mostly using the kind scripts from the upstream cilium repo.

**Comment by joestringer** [comment]
Probably this PR should be superseded or merged with https://github.com/cilium/cilium/pull/25217 . I think that cilium/cilium is the right place for most of these instructions.

File: README.md [hunk 2]
```diff
@@ -144,6 +148,51 @@ go run ./cmd/lvh run --image _data/images/base.qcow2
 go run ./cmd/lvh run --image _data/images/base.qcow2 --qemu-disable-kvm
 ```
 
+### Cilium development workflow
+
+Set up an LVH VM based on the kind image:
+
+```bash
+./scripts/pull_image.sh quay.io/lvh-images/kind:bpf-next-main
+lvh run --image $(ls -Art images/*.qcow2 | tail -n 1) --host-mount /path/to/cilium
```
Comment (joestringer) [inline README.md:157]: Now you can just do `lvh run --image quay.io/lvh-images/kind:bpf-next-main --host-mount /path/to/cilium`. No need to separately pull.

**Comment by joestringer** [comment]
No need for this PR any more, Cilium official docs have usage instructions with LVH.

**Pull request closed without merging**